ui/src/widgets/ui-button-group/UIButtonGroup.vue

```diff
@@ -73,7 +73,7 @@ export default {
             this.selection = msg.payload
         },
         onChange (value) {
-            if (value) {
+            if (value !== null && value !== undefined) {
```

Comment:
```suggestion
            if (value !== null && typeof value !== 'undefined') {
```

Though rare, `undefined` can be redefined (i've seen it)
 
Tis always safer to use `typeof` for testing `undefined`.